media-api/app/controllers/MediaApi.scala

```diff
@@ -148,8 +148,7 @@ object MediaApi extends Controller with ArgoHelpers {
           canUserDeleteImage(request, source) map { canDelete =>
             if (canDelete) {
               Notifications.publish(Json.obj("id" -> id), "delete-image")
-              // TODO: use respond
-              Accepted.as(ArgoMediaType)
+              Accepted
```

Comment:
I was thinking that the fact that the image won't be completely deleted until the message is consumed would be the reason, but realise that that's actually coupling the systems... never mind me.